python/orca/example/learn/pytorch/brainMRI/README.md

```diff
@@ -1,59 +1,72 @@
 # PyTorch BrainMRI example
-We demonstrate how to easily run synchronous distributed PyTorch training using PyTorch Estimator of Project Orca in Analytics Zoo. We use a simple Unet model to train on BrainMRI Segmentation dataset, which is a dataset for image segmentation. See [here](https://www.kaggle.com/s0mnaths/brain-mri-unet-pytorch/notebook) for the original single-node version. A lot of thanks to S0MNATHS, the orginal author of the code.
+We demonstrate how to easily run synchronous distributed PyTorch training using PyTorch Estimator of Project Orca in [BigDL](https://github.com/intel-analytics/BigDL). We use a simple Unet model to train on BrainMRI Segmentation dataset, which is a dataset for image segmentation. See [here](https://www.kaggle.com/s0mnaths/brain-mri-unet-pytorch/notebook) for the original single-node version.
 
 
-## Trouble Shooting
-- Difficult to identify a customized loss function; for example, dice coefficient
-    - Require a loss function creator to pass a customized loss function into orca
-- Unable to see the plot of matplotlib
-- Need to implement function to download dataset if it's not included in pytorch datasets
-
-## TODO
-- [ ] bigdl backend development
-- [ ] yarn development
-
 ## Prepare the environment
 
 We recommend you to use Anaconda to prepare the environment, especially if you want to run on a yarn cluster:
 
 ```
-conda create -n zoo python=3.7  # "zoo" is conda environment name, you can use any name you like.
-conda activate zoo
+conda create -n bigdl python=3.7  # "bigdl" is conda environment name, you can use any name you like.
+conda activate bigdl
 pip install torch
 pip install torchvision
+pip install albumentations
+pip install scikit-learn
+pip install opencv-python
 pip install matplotlib
 
-# For bigdl backend:
-pip install analytics-zoo  # 0.10.0.dev3 or above
-pip install jep==3.9.0
-pip install six cloudpickle
-
 # For torch_distributed backend:
-pip install analytics-zoo[ray]  # 0.10.0.dev3 or above
+pip install bigdl-orca[ray]
 
-# For spark backend
+# For spark backend:
 pip install bigdl-orca
 ```
 
-## Run on local after pip install
-
-`bigdl` backend is still developing.
+## Prepare the dataset
+You can download the dataset [here](https://www.kaggle.com/datasets/mateuszbuda/lgg-mri-segmentation/download). Put the downloaded data in the current working directory.
 
-You can run with `torch_distributed` backend via:
 
 ```
-python brainMRI.py --backend torch_distributed
+unzip archive.zip
+
+# if you use yarn-client, the dataset should be sent to the excutors. 
+# So we need zip the dataset as `.zip` file, then the program will send it to all excutors automatically. 
```

Comment:
If you run in yarn-client mode, you need to zip the images under kaggle_3m to the current working directory as well.